client/src/main/java/io/pravega/client/batch/StreamInfo.java

```diff
@@ -10,17 +10,24 @@
 package io.pravega.client.batch;
 
 import com.google.common.annotations.Beta;
-import io.pravega.client.stream.Stream;
+import io.pravega.client.stream.StreamCut;
 import lombok.Data;
 
+/**
+ * This class is used to represent Stream information. It currently includes
+ *  - scope of stream.
+ *  - name of stream.
+ *  - {@link StreamCut} which represents the current TAIL of the stream.
+ *
+ *  In future this will include the following:
+ *  - isSealed
+ *  - length of the stream.
+ *  - creation time of the stream.
+ */
 @Beta
 @Data
 public class StreamInfo {
```

Comment:
Looking at the values before it had a start and end time. I think we could have each of those as well as a start and end streamCut.